rdf-delta-integration-tests/src/main/java/org/seaborne/delta/systemtest/Matrix.java

```diff
@@ -148,12 +146,6 @@ public static void teardown() {
         zkServer = null;
     }
 
```

Comment:
The following code was never called anywhere, and since it used a `CuratorFramework`, I deleted it to prevent it from ever getting called in the future rather than updating it to use `ZkConnection`.